hands_on/fetch_merge.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from exercise_utils.cli import run_command
+from exercise_utils.gitmastery import create_start_tag
+
+__requires_git__ = True
+__requires_github__ = True
```

Comment:
If you're not using `gh`, you don't need this to be set to true
```suggestion
__requires_github__ = False
```